src/main/java/DukeEnumExceptions.java

```diff
@@ -0,0 +1,105 @@
+public enum DukeEnumExceptions {
```

Comment:
I really like your use of enum to handle Exceptions specific to each enum instance!